Cargo.toml

```diff
@@ -10,9 +10,14 @@ repository = "https://github.com/calavera/cargo-lambda"
 keywords = ["cargo", "subcommand", "aws", "lambda"]
 
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
+# For creating zip files.
```

Comment:
can you keep the dependencies sorted alphabetically? sorry for asking this, it just makes my life easier 😆

Reply:
Resolved.